Include/cpython/import.h

```diff
@@ -32,6 +32,7 @@ struct _frozen {
     const char *name;                 /* ASCII encoded string */
     const unsigned char *code;
     int size;
+    bool is_package;
```

Comment:
Changing the structure of `PyImport_FrozenModules` (specifically `struct _frozen`) and the meaning of the `size` field will break anyone freezing modules currently.  The fix to freeze.py will probably deal with that sufficiently, but there will always be someone out there that you don't know about who is using the feature you're changing. :/

So please add a "What's New" entry for this change.  Also, perhaps set `is_package` to true if `size` is negative.  (See below.)

Reply:
(Note that we changed that structure for deep-freezing anyways.)

Reply:
I made it backwards compatible so no need to add it to What's new.

Reply:
I'd recommend it as we want folks that do their own frozen modules to start doing it the new way, and it would be easy to miss.  It doesn't need to be a length entry.

Then again, updating the docs entry might be sufficient.  That should definitely happen either way.  See https://docs.python.org/3.11/c-api/import.html?highlight=frozen#c._frozen.

Reply:
Ok I added `versionchanged` to the c api docs.